src/ceph-daemon

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
@liewegas  this breaks el8 build. see https://fedoraproject.org/wiki/Changes/Make_ambiguous_python_shebangs_error

```
+ /usr/lib/rpm/redhat/brp-mangle-shebangs
mangling shebang in /usr/lib64/ceph/ceph-monstore-update-crush.sh from /usr/bin/env bash to #!/usr/bin/bash
mangling shebang in /usr/bin/ceph-post-file from /usr/bin/env bash to #!/usr/bin/bash
mangling shebang in /usr/bin/rbd-replay-many from /usr/bin/env bash to #!/usr/bin/bash
mangling shebang in /usr/bin/rbdmap from /usr/bin/env bash to #!/usr/bin/bash
mangling shebang in /usr/sbin/mount.fuse.ceph from /usr/bin/python3 to #!/usr/libexec/platform-python
*** ERROR: ambiguous python shebang in /usr/sbin/ceph-daemon: #!/usr/bin/env python. Change it to python3 (or python2) explicitly.
error: Bad exit status from /var/tmp/rpm-tmp.7BBpBi (%install)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.7BBpBi (%install)
+ rm -fr /tmp/install-deps.31095
```

https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos8,DIST=centos8,MACHINE_SIZE=huge/32254//consoleFull

can we make ceph-daemon package rely on python3?  is CR repo acceptable from user's perspective if they want to stick to pre RHEL/CentOS 7.7?

Reply:
I'm always wondering how to mark a script to be compatible to 2 and 3 without being a python package

Reply:
I thought I fixed this in the PR that merged earlier, but somehow the commit got missed.  See #31318 which should work around this